src/elisa/infer/samplers/ns/dynesty.py

```
@@ -0,0 +1,173 @@
+    def __init__(
+        self,
+        numpyro_model: Callable,
+        model_args: tuple = (),
+        model_kwargs: dict | None = None,
+        seed: int = 42,
+        ignore_nan: bool = False,
+        dynamic: bool = False,
+        **kwargs: dict,
+    ):
+        if ignore_nan:
+            warnings.warn(
```

Ruff (0.15.7)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)